modules/developer_manual/pages/core/unit-testing.adoc

```diff
@@ -78,17 +80,19 @@ available.
 
 Testing apps
 
-* To run test for a specific app with the provided PHPUnit version,
-change into `<webroot>/apps/<appnname>/<testfolder>` and call
-`<webroot>/lib/composer/phpunit/phpunit/phpunit` plus optional
-parameters when needed.
+* To run the tests for a specific app with the provided PHPUnit version,
```

Comment:
This change is a backport of words from #661 - which is "a good thing"
Note: the actual command below is correct for 10.0 `make test-php`
In #661 in `master` the command is `make test-php-unit` which is the correct command for "current" `stable10` after `10.1`